public/styles/interface.css

```diff
@@ -58,6 +59,43 @@ input {
   border: none;
 }
 
+#role {
+  float: left;
+  width: 20%;
+}
+
+#name {
+  display: flex;
+  justify-content: center;
+  font-size: 20px;
+  text-decoration: underline;
+}
```

Comment:
I'll trust you on the rest of the CSS. Maybe @nmatkins can review this better than I can.